### ISSUE TITLE
don't wrap last array item when trailingComma is 'none'

### ISSUE THREAD

**Issue by artembelik** [electrovir/prettier-plugin-newline-arrays#41]
Hi, thanks for the awesome plugin, but I noticed a strange behavior when trailingComma is 'none'

.prettierrc
```
{
    printWidth: 120,
    tabWidth: 4,
    useTabs: false,
    singleQuote: true,
    trailingComma: 'none', // <---
    plugins: ['prettier-plugin-multiline-arrays'],
}
```

source:
```js
const { join } = require('path');

const arr = [
    join('...', '...'),
    join('...', '...')
]; // <---
```

after `prettier --write`:
```js
const { join } = require('path');

const arr = [
    join('...', '...'),
    join('...', '...')]; // <---
```

**Comment by electrovir** [comment]
Nice catch! I'll try to repro and fix.

**Comment by electrovir** [comment]
Note: this also happens in JSON (since JSON doesn't allow trailing commas)

**Comment by adiroiban** [comment]
I have the same issue with `trailingComma: 'es5'`

Original code
```
{
    "files": [],
    "references": [
        {"path": "./tsconfig.app.json"},
        {"path": "./tsconfig.node.json"}
    ]
}
```

after this plugin is executed, I ended up with

```
{
    "files": [],
    "references": [
        {"path": "./tsconfig.app.json"},
        {"path": "./tsconfig.node.json"}]
}
```

**Comment by electrovir** [comment]
fixed for the next release

**Comment by electrovir** [comment]
released in [v4.0.0](https://www.npmjs.com/package/prettier-plugin-multiline-arrays/v/4.0.0)
